linera-views/tests/operations_tests.rs

```diff
@@ -138,3 +136,31 @@ async fn test_ordering_memory_specific() {
     ];
     test_ordering_keys_key_value_vec(key_value_operation, key_value_vec).await;
 }
+
+#[tokio::test]
+async fn test_big_value_read_write() {
+    use rand::{distributions::Alphanumeric, Rng};
+    let context = create_test_context();
+    for count in [50, 1024] {
+        let rng = rand::rngs::StdRng::seed_from_u64(2);
```

Comment:
We could use the faster `SmallRng` for tests where cryptographic security doesn't matter.

Reply:
The `SmallRng` is in version 0.8.5 but not 0.7.3
The upgrade is actually not straightforward.

Reply:
It should be in 0.7.3, too: https://docs.rs/rand/0.7.3/rand/rngs/struct.SmallRng.html
(Not a blocker, of course!)